runtime/gc_base/GCExtensions.hpp

```diff
@@ -87,8 +87,12 @@ class MM_IdleGCManager;
  */
 class MM_GCExtensions : public MM_GCExtensionsBase {
 private:
-	MM_OwnableSynchronizerObjectList* ownableSynchronizerObjectLists; /**< The global linked list of ownable synchronizer object lists. */
-	MM_ContinuationObjectList* continuationObjectLists; /**< The global linked list of continuation object lists. */
+	MM_OwnableSynchronizerObjectList* _ownableSynchronizerObjectLists; /**< The global linked list of ownable synchronizer object lists. */
+	MM_ContinuationObjectList* _continuationObjectLists; /**< The global linked list of continuation object lists. */
+#if defined(J9VM_GC_REALTIME)
+	MM_ReferenceObjectList* _referenceObjectLists; /**< A global array of lists of reference objects (i.e. weak/soft/phantom) */
+#endif /* J9VM_GC_REALTIME */
+	MM_UnfinalizedObjectList* _unfinalizedObjectLists; /**< The global linked list of unfinalized object lists. */
```

Comment:
Let's use the other style ('*' next to variable, not next to the type)  for any ObjectLists declarations/APIs code that you've touched in this PR.

I know, it may create some style inconsistencies in files you've touched, but hopefully we will fix them too at some point.